src/Storage/Controllers/InstancesController.cs

```diff
@@ -185,7 +186,24 @@ public class InstancesController : ControllerBase
 
                 (string person, string orgNo) = InstanceHelper.SeparatePersonAndOrgNo(instanceOwnerIdType, instanceOwnerIdValue);
 
-                instanceOwnerPartyId = await _registerService.PartyLookup(person, orgNo);
+                try
+                {
+                    instanceOwnerPartyId = await _registerService.PartyLookup(person, orgNo);
+                }
+                catch (Exception ex)
```

Comment:
If we decide to keep the exception I would like the catch to be specific.
```suggestion
                catch (PartyNotFoundException partyNotFoundException)
```